src/browser/modules/Stream/FrameTitlebar.jsx

```diff
@@ -60,11 +62,14 @@ import {
 } from 'shared/services/exporting/imageUtils'
 import {
   stringifyResultArray,
-  transformResultRecordsToResultArray
+  transformResultRecordsToResultArray,
+  recordToJSONMapper
 } from 'browser/modules/Stream/CypherFrame/helpers'
 import { csvFormat } from 'services/bolt/cypherTypesFormatting'
 import arrayHasItems from 'shared/utils/array-has-items'
 
+const JSON_EXPORT_INDENT = 2
```

Comment:
@oskarhane any suggestions for where to put this?

Reply:
No need to extract this at all imo.